docs/install.md

```diff
@@ -74,3 +74,32 @@ The different supported modes can be set by using the `hook-mode` option:
 - `podinformer`: Use a Kubernetes controller to get information about new pods. This option is racy and the first events produced by a container could be lost. This mode is selected when `auto` is used and the above modes are not available.
 - `ldpreload`: Adds an entry in `/etc/ld.so.preload` to call a custom shared library that looks for `runc` calls and dynamically adds the needed OCI hooks to the cointainer `config.json` specification. This feature only works when runc is used. Since this feature is highly experimental, it'll not be considered when `auto` is used.
 - `nri`: Use the [Node Resource Interface](https://github.com/containerd/nri). It requires containerd v1.5 and it's not considered when `auto` is used.
+
+## Required Kernel Versions
+
+This section summarizes the kernel versions and features that are required to
+run the gadgets. This has been generated by testing different Ubuntu versions
+and their shipped kernels, hence it's possible that some gadgets works in older
+kernels than the one mentioned here.
+
+
+| Gadget                 | Tools Mode | Required Kernel                 | BTF Required |
+|------------------------|------------|---------------------------------|--------------|
+| bindnsoop              | standard   | 4.15                            | N            |
+| bindnsoop              | core       | 5.4<sup>[1](#btfhubnote)</sup>  | Y            |
+| capabalities           |            | 4.15                            | N            |
+| dns                    |            | 5.4                             | N            |
+| execsnoop              | standard   | 4.15                            | N            |
+| execsnoop              | core       | 5.4<sup>[1](#btfhubnote)</sup>  | Y            |
+| network policy advisor |            |                                 |              |
```

Comment:
Did you have any kernel version for this one?

Reply:
I haven't gathered the data for that one. I just also realized I'm missing the seccomp policy advisor too.

Reply:
I think it's ok to merge this first version and then improve with the missing information in  another PR.